scss/_variables.scss

```diff
@@ -371,22 +386,22 @@ $table-border-level:          -6 !default;
 //
 // Shared variables that are reassigned to `$input-` and `$btn-` specific variables.
 
-$input-btn-padding-y:         .375rem !default;
-$input-btn-padding-x:         .75rem !default;
+$input-btn-padding-y:         if($enable-responsive-font-sizes, .375em, .375rem) !default;
+$input-btn-padding-x:         if($enable-responsive-font-sizes, .75em, .75rem) !default;
```

Comment:
Found a solution for this, but this will cause nested `calc()` functions. To fix this, I've added `postcss-calc` to fix this (a new dependency in our build, I know).

An alternative is to store all calculations that are done in variables. This will introduce about 5 new variables with complex calculation and will also increase the complexity of the `$input-height` (and sizes). I don't know if that's worth it.

Reply:
I noticed `postcss-calc` also trims some spaces from the examples where custom properties I used. This PR https://github.com/twbs/bootstrap/pull/27634 fixes this by replacing the custom properties with the values (IE doesn't support them anyway).

Reply:
Just removed `postcss-calc`